docs/guide/tutorial-docker.md

```diff
@@ -0,0 +1,50 @@
+Yii and Docker
+==============
+
+For development and deployments Yii applications can be run as Docker containers.
+
+You can read more about Docker containers on [docker.com](https://www.docker.com/what-docker).
+
+## Requirements
+
+- `docker`
+- `docker-compose`
+
+Visit the [download page](https://www.docker.com/community-edition) for Docker tools.
+
+## Resources
+
+PHP-base images for Yii can be found at [yii2-docker](https://github.com/yiisoft/yii2-docker).
+
+
+*TBD:* Docker support for [yii2-app-advanced](https://github.com/yiisoft/yii2-app-advanced/pull/347).
```

Comment:
Docker is supported in basic project template. Support for advanced project template is pending.